addon/lib/Feature.jsm

```diff
@@ -979,6 +1162,7 @@ class Feature {
   }
 
   async uninit() {
+    await this.reportBehaviorSummary();
```

Comment:
Since I call `uninit` on Bootstrap shutdown, this will get called anytime Firefox shuts down, not just at the very end... Consider calling `this.feature.reportBehaviorSummary()` in bootstrap?

```javascript
// in bootstrap.shutdown()...
if (isUninstall) {
  // other stuff
  await this.feature.reportBehaviorSummary();
  await Storage.clear();  // will have to import Storage.jsm into bootstrap.js
}
```

Reply:
I intentionally added it to uninit() to make sure it's still reported even if telemetry fails on shutdown due to some race condition or another issue. Your call though. From an analysis standpoint, it's easier to have only one ping from each user, but it's not that difficult to only find the latest ping either.

Reply:
Okay thanks for that insight! I tested it in a bunch of different uninstall cases, and it seems to get sent properly. I'll leave it this way for now.